src/SFA.DAS.EAS.Account.Worker/Infrastructure/JobLogger.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.IO;
+
+namespace SFA.DAS.EAS.Account.Worker.Infrastructure
+{
+	static class JobLogger
```

Comment:
Done

It would be nice if we could create a log per job (just so we get the type in Kibana as the job type) but this is a bit tricky to do as an *instance* of the TraceWriter must be registered globally with the jobhost so we can't link it directly to an IJob instance and we don't have a 'job context'.